core/metrics-core-impl/src/main/java/org/hawkular/metrics/core/impl/MetricsServiceImpl.java

```diff
@@ -103,7 +100,7 @@
         private final MetricId metricId;
 
         public DataRetentionKey(String tenantId, MetricType type) {
-            metricId = new MetricId(tenantId, type, "[" + type.getText() + "]");
+            metricId = new MetricId(tenantId, type, "$" + type.getText());
```

Comment:
If $ is suddenly a magic key (repeated in several places), shouldn't it be a some sort of constant instead of magic-char in the strings?